pkgs/development/python-modules/clickgen/default.nix

```
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pillow
+, libX11
+, libXcursor
+, libpng
+, python
+}:
+
+buildPythonPackage rec {
+  pname = "clickgen";
+  version = "1.1.9";
+
+  src = fetchFromGitHub {
+    owner = "ful1e5";
+    repo = "clickgen";
```

```suggestion
    repo = pname;
```

Minor nitpick, but repo has the same name as package.

---

If @AdsonCicilioti wants to do this feel free, however I don't care too much about this change (as you said, this is a nitpick).

---

> 

I think this shouldn't be dynamic, despite being a simple name, upstream can change its default to capitalized for example. e Believed that simple name nixpkgs would always be lowercase. And if there's no pattern on both sides then I don't see much of it.